server/core/store.go

```diff
@@ -390,6 +390,21 @@ func (s *StoreInfo) GetUptime() time.Duration {
 	return 0
 }
 
+// GetMaxScore return the maxScore
+func (s *StoreInfo) GetMaxScore() float64 {
+	return s.maxScore
+}
+
+// SetMaxScore is used to set maxScore
+func (s *StoreInfo) SetMaxScore(maxScore float64) {
+	s.maxScore = maxScore
+}
+
+// CalculateMaxScore is used to calculate max score according to capacity.
+func (s *StoreInfo) CalculateMaxScore() float64 {
+	return float64(s.GetCapacity() + 10*1024*1024)
```

Comment:
Can we config the value?

Reply:
Of course. But what is name? how about `flexible-score`?